tools/code_format/check_format.py

```diff
@@ -150,6 +150,7 @@
 COMMENT_REGEX = re.compile(r"//|\*")
 DURATION_VALUE_REGEX = re.compile(r'\b[Dd]uration\(([0-9.]+)')
 PROTO_VALIDATION_STRING = re.compile(r'\bmin_bytes\b')
+# e.g., "category: lower-case word."
```

Comment:
Would it make more sense to update the error message to explain better?

Reply:
I'm not really familiar with any of this infrastructure, just added this comment after tripping on the presubmit, so I'd rather not fix the error message here.

Reply:
sure, but if you're adding a comment, everyone after you is going to deal with the same confusing error message.  Better to change that?

        if not match:
          reportError("Version history line malformed. "
                      "Does not match VERSION_HISTORY_NEW_LINE_REGEX in check_format.py\n %s" %
                      line)

You could change the text to something like
 
Does not match VERSION_HISTORY_NEW_LINE_REGEX in check_format.py\n %s\nPlease use messages in the form 'category: feature explanation, starting with lower-cased letter and ending with a period.'"

or something?